### PR TITLE
fix(daterangepicker): prevent DRP sticky behavior

### PR DESCRIPTION
**Pull request by SashaBayan**

### CODE REVIEW

**Comment by SashaBayan** [comment]
I am happy to change this. @ianwremmel how are you envisioning we use props on a page by page basis to dynamically adjust the DRP to be attached to the correct component?

**Comment by ianwremmel** [comment]
The autocomplete component passes values from data attributes to vue props, but I think we want to be cautious about adhering too closely to drp’s attributes given I do t see that jquery plugin as being a long term solution. 

For this particular instance, I think you should be able to use [data-press-component=daterangepicker] as the parent and just always assume that.

File: src/lib/css-selector.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/**
+ * Converts an HTML attribute string to a CSS selector
+ * by pre-pending each word in a class string with a period
+ * e.g. "one two three" => ".one.two.three';
+ * @param {string} attr
+ * @returns {string}
+ */
+export function attributeToClassSelector(attr) {
```
[the remark this answers is not in the document]
Comment (SashaBayan) [reply src/lib/css-selector.js]: I suppose we could do that. It not be very semantic, however, to add CSS selector syntax onto an HTML attribute though. 

When placed on the HTML tag, `data-parent-el` is a string like so:
```
<div data-parent-el="ui search etc" />
```
The `parentEl` property requires a CSS selector. So when we take the selector from the attribute, we have to convert it to `.ui.search.etc`, hence this utility. 

I'm open to other ways of doing this, but this gets the job done.

File: src/components/daterangepicker/index.js [hunk 2]
```diff
@@ -39,6 +40,8 @@ export default class DateRangePicker extends PressComponentBase {
     drp.setAttribute('v-model', baseModelName);
     drp.setAttribute('start-key', startAttrs.key);
     drp.setAttribute('end-key', endAttrs.key);
+    parentElAttrs && drp.setAttribute('parent-el', parentElAttrs);
```
[the remark this answers is not in the document]
Comment (SashaBayan) [reply src/components/daterangepicker/index.js]: We've used this a fair amount in FE land. It used to strike me as odd in the beginning, but I've come to like it.

**Pull request merged**